shanoir_downloader.py

```diff
@@ -44,6 +50,11 @@ def add_ids_arguments(parser):
 	parser.add_argument('-sbid', '--subject_id', default='', help='The subject id to download.')
 	return
 
+def add_shanoir2bids_common_arguments(parser):
```

Comment:
It's best if `shanoir_downloader.py` remains independent of `shanoir2bids.py` (so that we can read  `shanoir_downloader.py` without knowing of `shanoir2bids.py`).

In `shanoir2bids.py`, line 278 :
`shanoir_downloader.add_shanoir2bids_common_arguments(parser)`
should be
```

	shanoir_downloader.add_username_argument(parser)
	shanoir_downloader.add_output_folder_argument(parser, required=False)

```
that would be neat.